libs/core/src/components/pds-loader/readme.md

```
@@ -33,9 +33,9 @@
 
 ### Used by
 
-- [pds-button](../pds-button)
-- [pds-combobox](../pds-combobox)
-- [pds-multiselect](../pds-multiselect)
+ - [pds-button](../pds-button)
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)